Makefile

```diff
@@ -57,3 +57,9 @@ lint-fix: bin/golangci-lint ## Fix lint violations.
 .PHONY: generate
 generate: ## Run all generate-* jobs in bulk.
 	cd tools; go generate
+
+##@ New kubernetes control-plane patchversions
+
+.PHONY: update-patchversions
+update-patchversions: ## Run update-patchversion script to generate new PR.
```

Comment:
The name should be more specific, e.g., update-k8s-patch-versions.

Reply:
good name, i used it